rtgui/guiutils.h

```diff
@@ -430,6 +436,22 @@ class MyImageMenuItem : public Gtk::MenuItem
     const Gtk::Label* getLabel () const;
 };
 
+class MyProgressBar : public Gtk::ProgressBar
```

Comment:
The `guiutils` package is becoming quite large. What do you think about splitting it up, e.g. making a `mywidgets` subfolder of `rtgui` with separate headers or something like that?